test/test-maker-taxo.rb

```diff
@@ -31,7 +31,7 @@ def setup
             "http://dmoz.org/Computers/Data_Formats/Markup_Languages/SGML/",
             "http://dmoz.org/Computers/Programming/Internet/",
           ]
-         },
+        },
```

Comment:
```suggestion
         },
```

Not in `Style/StringLiterals`.